files/core/base.sh

```diff
@@ -21,7 +18,7 @@ install_file authorized_keys /root/.ssh/authorized_keys
 chmod 600 /root/.ssh/authorized_keys
 echo "StrictHostKeyChecking no" >> /root/.ssh/config
 
-yum -y install yum-plugin-priorities yum-utils
+yum -y install yum-utils
```

Comment:
You can take the yum-utils line out as well, install it as part of your client local repo setup script